Source/get_prediction.py

```diff
@@ -0,0 +1,96 @@
+import calendar
+import os
+import pickle  # noqa
+from collections import Counter
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from tensorflow.keras.models import load_model
+from tensorflow.keras.preprocessing.sequence import pad_sequences
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+
+def open_file(path):
+    with open(path, encoding="cp1252") as readfile:
+        df = pd.read_csv(readfile)
+    df = df[df["tweet"].notna()]
+    return df["tweet"], df["date"]
+
+
+def get_max_len(tweets):
```

Comment:
You can replace this function with https://docs.python.org/3/library/functions.html#max